applications/deployments/deployment-strategies.adoc

```diff
@@ -52,7 +52,15 @@ include::modules/deployments-rolling-strategy.adoc[leveloffset=+1]
 include::modules/deployments-canary-deployments.adoc[leveloffset=+2]
 include::modules/deployments-creating-rolling-deployment.adoc[leveloffset=+2]
 include::modules/odc-starting-rolling-deployment.adoc[leveloffset=+2]
+.Additional resources
+- xref:../applications/application-life-cycle-management/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Creating and deploying applications on {product-title} using the *Developer* perspective]
```

Comment:
None of these links work. I believe you need to back up two levels for these to work:

```
xref:../../applications/application-life-cycle-management/ ...
```

Recommend building the doc locally and testing (always, but especially when adding links, complex items, etc.)

Reply:
Will do. Apologies, I waited till the build passed.

Reply:
Yeah, I think @vikram-redhat said there might be an issue with the build not failing properly for something like this. You can see the error in the build output though: https://travis-ci.com/openshift/openshift-docs/builds/141800198

That's why I'd recommend getting in the habit of building locally to be sure.

Reply:
Yes, I apologize, I am not sure why these sorts of builds are not getting flagged as failing instead of getting the green check. I will investigate. 

@openshift/team-documentation @openshift/team-aligned-docs I have two such instances where build that failed didn't get the red cross of failure. If you come across any more could you please add to this Jira as a comment (https://issues.redhat.com/browse/OSDOCS-867)? 

FWIW, I had sent these examples to the Travis team thinking that it might be an issue in their build system but they were not able to find anything.